gui/network/forms.py

```diff
@@ -644,9 +644,13 @@ def clean_gc_netwait_ip(self):
 
     def clean(self):
         cleaned_data = self.cleaned_data
+        domains = cleaned_data.get("gc_domains")
         nameserver1 = cleaned_data.get("gc_nameserver1")
         nameserver2 = cleaned_data.get("gc_nameserver2")
         nameserver3 = cleaned_data.get("gc_nameserver3")
+        if len(domains.split()) > 6:
```

Comment:
You probably need to process that field so the entries are properly added to the database, without extra spaces and/or newlines:
`if domains:
    domains = domains.split()
    cleaned_data['gc_domains'] = '\n'.join(domains)
    if len(domains) > 5:
```

Not the `> 5`, you can have at max 6, you already have one from the main domain.